assignment-sweet-compose/answer/docker-compose.yml

```diff
@@ -56,11 +55,16 @@ services:
       - traefik.frontend.rule=Host:vote.localhost
 
   result:
-    build:
-      context: ../result
-    command: nodemon index.js
```

Comment:
There is no `index.js` file in the project

Reply:
Oops, it's server.js in the lastest code update a few months back.